### PR TITLE
Lowercased the search string, so case doesn't produce different ranking results

### PR DESCRIPTION
**Pull request by dploeger**
Fixes the issue, that came up in #1073 after merging

### CODE REVIEW

File: app/scripts/models/entry-model.js [hunk 1]
```diff
@@ -670,7 +670,7 @@ const EntryModel = Backbone.Model.extend({
         _.forEach(ranking, rankingEntry => {
             if (this._getFieldString(rankingEntry.field).toLowerCase() !== '') {
                 const calculatedRank = Ranking.getStringRank(
-                    searchString,
+                    searchString.toLowerCase(),
```
Comment (antelle) [inline app/scripts/models/entry-model.js]: Let's pass `textLower` in `rankComparator`, instead of doing `toLowerCase` for every entry.

Comment (dploeger) [reply app/scripts/models/entry-model.js]: Hmm... textLower doesn't reach the rankComparator. It is prepared here:

[https://github.com/keeweb/keeweb/blob/f0ff26ebb4e40f4c8728230b585f052feb471078/app/scripts/models/app-model.js#L285](https://github.com/keeweb/keeweb/blob/f0ff26ebb4e40f4c8728230b585f052feb471078/app/scripts/models/app-model.js#L285)

but the prepared filter including textLower is not persisted into this.filter, so it is lost again here:

[https://github.com/keeweb/keeweb/blob/f0ff26ebb4e40f4c8728230b585f052feb471078/app/scripts/models/app-model.js#L277](https://github.com/keeweb/keeweb/blob/f0ff26ebb4e40f4c8728230b585f052feb471078/app/scripts/models/app-model.js#L277)

Shouldn't the filter better be prepared here:
[https://github.com/keeweb/keeweb/blob/f0ff26ebb4e40f4c8728230b585f052feb471078/app/scripts/models/app-model.js#L267](https://github.com/keeweb/keeweb/blob/f0ff26ebb4e40f4c8728230b585f052feb471078/app/scripts/models/app-model.js#L267)

Comment (antelle) [reply app/scripts/models/entry-model.js]: Hm, isn't it passed here? https://github.com/keeweb/keeweb/blob/develop/app/scripts/util/comparators.js#L16

Comment (antelle) [reply app/scripts/models/entry-model.js]: Read it again, I see. Yes, I think it makes sense to move it there

**Comment by dploeger** [comment]
Done

File: app/scripts/models/entry-model.js (hunk 1 above)
Comment (dploeger) [reply app/scripts/models/entry-model.js]: Done

**Comment by antelle** [comment]
Thanks! 🎁

**Pull request merged**